src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEMethodSymbol.vb

```diff
@@ -1083,26 +1083,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
         End Property
 
         Friend Overrides Function GetUseSiteErrorInfo() As DiagnosticInfo
+            Dim result As DiagnosticInfo
+
             If Not _packedFlags.IsUseSiteDiagnosticPopulated Then
                 Dim errorInfo As DiagnosticInfo = CalculateUseSiteErrorInfo()
                 EnsureTypeParametersAreLoaded(errorInfo)
-                Return InitializeUseSiteErrorInfo(errorInfo)
-            End If
-
-            Dim uncommonFields = _uncommonFields
-            If uncommonFields Is Nothing Then
```

Comment:
Same. We won't be losing an optimization. As it will not be initialized if it was null.

---
In reply to: [119781690](https://github.com/dotnet/roslyn/pull/19658#discussion_r119781690) [](ancestors = 119781690)